flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/ScalarFunctionsTest.scala

```diff
@@ -889,6 +916,69 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
       "")
   }
 
+  @Test
+  def testChr(): Unit = {
+    testAllApis(
+      'f14.chr(),
+      "f14.chr()",
+      "CHR(f14)",
+      "null")
+
+    testAllApis(
+      'f34.chr(),
+      "f34.chr()",
+      "CHR(f34)",
+      "null")
+
+    testAllApis(
+      'f34.chr(),
+      "f34.chr()",
+      "CHR(f34)",
+      "null")
+
+    testAllApis(
+      'f36.chr(),
+      "f36.chr()",
+      "CHR(f36)",
+      "A")
+
+    testAllApis(
+      'f37.chr(),
+      "f37.chr()",
+      "CHR(f37)",
+      "a")
+
+    testAllApis(
+      'f38.chr(),
+      "f38.chr()",
+      "CHR(f38)",
+      "ÿ")
+
+    testAllApis(
```

Comment:
Duplicated

Reply:
There are three test cases that seem to be duplicates, but their SQL functions pass different input types for verification purposes.

Reply:
This is duplicated with a test case above.